x-pack/plugins/ml/public/util/index_utils.js

```diff
@@ -9,39 +9,39 @@
 import { notify } from 'ui/notify';
 import { SavedObjectsClientProvider } from 'ui/saved_objects';
 
-let indexPatterns = [];
+let _indexPatterns = [];
```

Comment:
maybe `indexPatternCache` would be a better name?

Reply:
I like it! Done.